app/modules/teams/parameters.py

```diff
@@ -11,7 +11,7 @@
 from .models import Team
 
 
-class CreateTeamParameters(PostFormParameters, schemas.BaseTeamSchema):
+class CreateTeamParameters(PostFormParameters, schemas.BaseTeamSchema):  # pylint: disable=too-many-ancestors
```

Comment:
Please, try to find a config for a number of max ancestors and increment it in `.pylintrc`, so we don't need to add this type of error everywhere.